pkg/operator/controller/ingress/deployment.go

```diff
@@ -1024,68 +1022,6 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 			clientAuthCAPath := filepath.Join(clientCAVolumeMount.MountPath, clientCABundleFilename)
```

Comment:
https://github.com/openshift/cluster-ingress-operator/pull/891/files#diff-03760f03eeac1174e38e1ec5fecadcab4cb38d05e00509f201d7ae64dc1c7c36L90 `RouterClientAuthCRL    = "ROUTER_MUTUAL_TLS_AUTH_CRL"` is unused